gateway-server/src/main/java/org/apache/knox/gateway/services/token/impl/DefaultTokenStateService.java

```diff
@@ -168,15 +167,15 @@ public boolean isExpired(final JWTToken token) {
 
   @Override
   public boolean isExpired(final String token) {
-    boolean isExpired;
+    boolean isUnknownToken;
```

Comment:
Should still be isExpired ?

Reply:
It's just a variable name change, I changed it because it was similar to the method name (nothing wrong with it but just a habit from other programming language)